src/main/webapp/linkWizard/SelectSampleConsent.js

```diff
@@ -109,7 +109,7 @@ export const SelectSampleConsent = hh(class SelectSampleConsent extends Componen
 
   loadOptions() {
     let documentOptions = [];
-    DOCUMENT_TYPE.forEach(type => {
+    CONSENT_DOCUMENTS.forEach(type => {
```

Comment:
This could be more efficient ... instead of mutating a variable, you can assign it to the result of the `map` operation, which already returns a collection of whatever is defined inside the map. Something like this: 
```
const documentOptions = CONSENT_DOCUMENTS.map(type => { value: type, label: type });
```
In general, try to [avoid mutation and instead use constants](https://medium.com/poka-techblog/simplify-your-javascript-use-map-reduce-and-filter-bd02c593cc2d).

Reply:
I agree, good suggestion. I'm going to do the same also in my actual story.